manager/controllers/systems.go

```diff
@@ -39,7 +39,9 @@ type SystemItemAttributes struct {
 	RheaCount      int        `json:"rhea_count" csv:"rhea_count" query:"system_platform.advisory_enh_count_cache"`
 	// Not used
 	//Enabled        bool       `json:"enabled" csv:"enabled" query:"(NOT system_platform.opt_out)"`
-	Stale bool `json:"stale" csv:"stale" query:"system_platform.stale"`
+	Stale     bool `json:"stale" csv:"stale" query:"system_platform.stale"`
```

Comment:
These values are **not** included in SystemDetail response. It should be also checked in the systems_test.go and advisory_systems_test.go.